### DISCUSSION TITLE
Dependencies

### DISCUSSION THREAD

**Discussion by hackatore** [ntsaini/node-red-contrib-function-npm#42] [General]
I'm running into these dependency issues when I upgrade node-red-contrib-function-npm.
Is there a graceful resolution?   

glob-parent  <5.1.2
Severity: high
Regular expression denial of service - https://github.com/advisories/GHSA-ww39-953v-wcq6
fix available via `npm audit fix --force`
Will install node-red-contrib-function-npm@0.3.0, which is a breaking change
node_modules/glob-parent
  fast-glob  <=2.2.7
  Depends on vulnerable versions of glob-parent
  node_modules/fast-glob
    npm-install-to  >=1.2.0
    Depends on vulnerable versions of fast-glob
    node_modules/npm-install-to
      node-red-contrib-function-npm  >=0.4.0
      Depends on vulnerable versions of npm-install-to
      node_modules/node-red-contrib-function-npm

**Comment by ntsaini** [comment]
Looks like it is complaining about vulnerability in some dependency, not sure why it is trying to install 0.3.0 the latest is 0.4.3. May be you can try installing the latest version by specifying the version identifier @0.4.3
